### ISSUE TITLE
New Install Question

### ISSUE THREAD

**Issue by hg2s** [bsc101/roon-extension-macroon-readme#1]
I just installed the extension on Roon 1.8 (1021) running on an Ubuntu 20.04 server. I fumbled a bit at first to get it going but I am pleased to report that it is up and running. 

Am I correct that it requires the application to be running (via terminal in my case) in order for the extension to be available in Roon?

**Comment by bsc101** [comment]
Yes, the application _is_ the extension, so it must be running to be available in roon. I added the application to "autostart applications" (Linux Mint in my case), so I don't have to run it manually after booting... Is that what you wanted to know?

**Comment by hg2s** [comment]
Yes, that's it exactly. Just wanted to make sure I'm on the right track before I delve in.
